pymatgen/entries/computed_entries.py

```diff
@@ -63,7 +225,16 @@ def __init__(self,
         """
         super().__init__(composition, energy)
         self.uncorrected_energy = self._energy
-        self.correction = correction
+        self.energy_adjustments = energy_adjustments if energy_adjustments else []
+
+        if correction:
```

Comment:
I suggest this instead.
``` python
        if correction > 0.0 and energy_adjustments is not None:
```

Reply:
But then I would miss the second part of the logic: if the correction >0 and energy_adjustments IS none, we have to set the correction:

```
if correction >0.0:
            if energy_adjustments is not None:
                raise ValueError("Argument conflict! Setting correction = {:.3f} conflicts "
                                 "with setting energy_adjustments. Specify one or the "
                                 "other.".format(correction))

            self.correction = correction
```